src/FsAutoComplete/CodeFixes.fs

```diff
@@ -14,11 +14,17 @@ module LspTypes = LanguageServerProtocol.Types
 module Types =
   type IsEnabled = unit -> bool
 
+  type FixKind =
```

Comment:
@Krzysztof-Cieslak thought we might need this for this effort, but I'm not sure we will end up needing it. It's still useful to have, as it models a subset of the domain that's interesting.